### ISSUE TITLE
UnusedUses: Strange false positive in multiline indented annotations with constants

### ISSUE THREAD

**Issue by Majkl578** [slevomat/coding-standard#283]
This piece of code incorrectly reports unused Y:
```php
<?php

namespace Test;

use Y;

class Foo
{

    /**
     * @X(
     *     a=Y::SOME,
     *     b={
     *         @Z(
     *             code=123
     *         )
     *     }
     * )
     */
	public function bar()
	{
	}
}
```
but when I remove the newline & indent before completely unrelated `@Z`, no error is reported:
```php
<?php

namespace Test;

use Y;

class Foo
{

    /**
     * @X(
     *     a=Y::SOME,
     *     b={@Z(
     *             code=123
     *         )
     *     }
     * )
     */
	public function bar()
	{
	}
}
```

```
 5 | ERROR | [x] Type Y is not used in this file. (SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse)
```

dev-master @ 09b4ce5

**Comment by kukulich** [comment]
In the first example the “@Z” is probably detected as T_DOC_COMMENT_TAG. Parsing of annotations is broken in PHPCS. I tried to fixed it in AnnotationHelper but it looks I did not fixed all situations. However it should be fixable relatively easy with your perfect test case!

**Comment by kukulich** [comment]
Should be fixed via https://github.com/slevomat/coding-standard/commit/946e8349addd4b37df97df64941e0e648cb035d8
